DataRepo/loaders/msruns_loader.py

```
@@ -1659,17 +1684,22 @@ def parse_mzxml(cls, mzxml_path, full_dict=False):
             mzxml_path_obj = Path(mzxml_path)
 
         if not mzxml_path_obj.is_file():
+            errs_buffer.buffer_error(FileNotFoundError(f"File not found: {mzxml_path}"))
```

C. This was silently failing before.  So we need to buffer an actual error about file not found.